home-work-2/style.css

```
@@ -0,0 +1,185 @@
+html {
+    height: 100%;
+}
+
+.body {
+    background-color: #f9f9f9;
+}
+
+.container {
+    margin: auto;
+    height: 100%;
+    width: 1060px;
+    background-color: #fff;
+    -webkit-box-shadow: 0px 5px 5px 0px rgba(0,0,0,0.3);
+    -moz-box-shadow: 0px 5px 5px 0px rgba(0,0,0,0.3);
+    box-shadow: 0px 5px 5px 0px rgba(0,0,0,0.3);
+    padding: 40px;
+    box-sizing: border-box;
+}
+
+/*HEADER*/
+
+#logo {
+    padding-bottom: 40px;
+}
+
+#main-menu li{
+    color: #fff;
+    padding: 0 20px;
+    display: table-cell;
+    vertical-align: middle;
+    height: 100%;
+}
```

u can add transparent border on this lvl (to have same width on hover)
```suggestion
    border-left: 1px solid transparent;
    border-right: 1px solid transparent;
}
```

---

fixed